src/Week2_CodingTasks/printConsecutiveNumberTask.java

```diff
@@ -0,0 +1,59 @@
+package Week2_CodingTasks;
+
+import java.util.Scanner;
+
+public class printConsecutiveNumberTask {
```

Comment:
The code is good, but let's rename the file to TASK3_Zee (instead of printConsecutiveNumbetTask) so we hav the same pattern for everyone